source/gamemodes/BoostMod.cpp

```diff
@@ -0,0 +1,138 @@
+// gamemodes/BoostMod.cpp
+// A general team based boost modifier gamemode for Rocket Plugin.
+// Credits to Stanbroek (rocket plugin author) 
+// and ubelhj (author of SpeedBoost plugin, source of some of the code here).
+// 
+// Author:       Al12
+// Version:      0.1.0 17/07/20
```

Comment:
Normally I set the version to the Rocket Plugin version that the game mode was made in,
but I think giving game modes their own version is better.